src/testUtils/TestRig.react.js

```diff
@@ -28,6 +28,8 @@ export default class TestRig {
 
     this.screwOn = this.boltOn
     this.screwOff = this.boltOff
+    this.toggleCheckbox = this.toggleElement.bind(this)
```

Comment:
why do these need binding but none of the other helpers do 

(this isn't a react component, not sure if that was just habit?)